app/scripts/models/reliers/oauth.js

```diff
@@ -93,6 +93,13 @@ var OAuthRelier = Relier.extend({
           this._setupSignInSignUpFlow();
         }
 
+        // A lot of OAuth clients do not specify an entrypoint, which makes
+        // keeping track of where users are signing up from more difficult than
+        // need be. If no entrypoint is specified, use the client_id. See #6757
+        if (! this.has('entrypoint')) {
+          this.set('entrypoint', this.get('clientId'));
```

Comment:
small warning here is that @davismtl has to track down these client ids and match them with names. However, I guess this is better than `none` ?

Reply:
But we could just use the existing translation on the back-end instead.